onnxruntime/contrib_ops/cuda/transformers/generation_cuda_impl.cu

```diff
@@ -805,6 +806,189 @@ void UpdateDecoderMaskedMultiHeadAttentionCacheIndirection(int32_t* tgt_indir_ca
                                                                                           max_seq_length,
                                                                                           current_length);
 }
+
+#ifndef USE_ROCM
+namespace {
+template <typename T, size_t size>
+struct TypeMapper : public V_vec_m_<T, size> {};
```

Comment:
>V_vec_m_ [](http://example.com/codeflow?start=27&length=8)

It is better to expand the V_vec_m_ directly

Reply:
currently the int mapper only applies to this case